### PR TITLE
Add a needs-cherry-pick label 🏷

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

This should be useful for project to mark pull requests to be
cherry-pick in the latest release branch (for next minor release).

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

/cc @afrittoli 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: label_sync/labels.yaml [hunk 1]
```diff
@@ -223,6 +223,12 @@ default:
       target: prs
       prowPlugin: needs-rebase
       addedBy: prow
+    - color: 7057ff
```
Comment (afrittoli) [inline label_sync/labels.yaml]: This should be cherry-colour

Comment (vdemeester) [reply label_sync/labels.yaml]: ahah I like it

Comment (afrittoli) [reply label_sync/labels.yaml]: thank you! 🙏

**Pull request merged**